kubernetes/platform-services/Chart.yaml

```diff
@@ -28,37 +28,50 @@ dependencies:
     version: 0.1.7
   - name: span-normalizer
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.5.3
+    version: 0.5.4
   - name: raw-spans-grouper
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.5.3
+    version: 0.5.4
   - name: hypertrace-trace-enricher
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.5.3
+    version: 0.5.4
   - name: hypertrace-view-generator
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.5.3
+    version: 0.5.4
   - name: hypertrace-ui
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
     version: 0.69.3
   - name: hypertrace-graphql-service
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
     version: 0.7.3
-  - name: attribute-service
```

Comment:
Why only this has been removed? I think, for this release, it will be disabled.

Reply:
He added it below.